acceptance/bundle/templates/experimental-jobs-as-code/output/my_jobs_as_code/pyproject.toml

```diff
@@ -29,7 +30,7 @@ where = ["src"]
 [tool.uv]
 ## Dependencies for local development
 dev-dependencies = [
-    "databricks-bundles==x.y.z",
+    "databricks-bundles==0.251.0",
```

Comment:
Not totally sure what happens during the release. Is there a window where we have this test failing? @kanterov @pietern

Reply:
Not sure either. @kanterov, can you comment?

Reply:
I think for simplicity we need to keep sed part that we removed in this PR

Reply:
Can we just run 'go test -update' for this test?

Reply:
I think it's a bit dangerous to run "go test -update" during release because we don't review diffs

Reply:
Fair point. We don't need sed though: https://github.com/databricks/cli/pull/2880